website/docs/en/plugins/rspack/css-extract-rspack-plugin.mdx

```diff
@@ -169,8 +169,40 @@ module.exports = {
       {
         test: /\.css$/i,
         use: [rspack.CssExtractRspackPlugin.loader, 'css-loader'],
+        type: 'javascript',
       },
     ],
   },
 };
 ```
+
+:::info
+Please note when opening built-in CSS support. (`experiments.css`), Files ending with `.css` will be automatically treated as `css` modules. If you want to use this plugin, make sure that the rule types with `CssExtractRspackPlugin.loader` set are all overridden by `javascript` instead of the default `css`.
```

Comment:
```suggestion
Please note when opening built-in CSS support. (`experiments.css`), Files ending with `.css` will be automatically treated as `css` modules. If you want to use this plugin, make sure that the rule types with `CssExtractRspackPlugin.loader` set are all overridden by `javascript/auto` instead of the default `css/auto`.
```